src/WaCore.Crud.Contracts/Services/IWacListDataService.cs

```diff
@@ -8,6 +8,10 @@ namespace WaCore.Crud.Contracts.Services
 {
     public interface IWacListDataService<TFilter, TDto>
     {
+        TDto Get(object id);
+        Task<TDto> GetAsync(object id);
+        IPagedList<TDto> GetAll(TFilter filter);
```

Comment:
I don't like the name GetAll. If it takes a filter object, it means it doesn't always return ALL records. I'd call it GetList.

Reply:
Renamed to GetList